features/built_in_matchers/respond_to.feature

```diff
@@ -92,3 +92,185 @@ Feature: `respond_to` matcher
       | expected 7 not to respond to :zero? with 0 arguments    |
       | expected 7 to respond to :between? with 7 arguments     |
       | expected 7 not to respond to :between? with 2 arguments |
+
+  @skip-when-splat-args-unsupported
+  Scenario: specify arguments range
+    Given a file named "respond_to_matcher_argument_range_checking_spec.rb" with:
+      """ruby
+      class MyClass
+        def build(name, options = {})
+
+        end
+
+        def inspect
+          'my_object'
+        end
+      end
+
+      RSpec.describe MyClass do
+        it { is_expected.to respond_to(:build).with(1..2).arguments }
+        it { is_expected.not_to respond_to(:build).with(0..1).arguments }
+        it { is_expected.not_to respond_to(:build).with(2..3).arguments }
+        it { is_expected.not_to respond_to(:build).with(0..3).arguments }
+
+        # deliberate failures
+        it { is_expected.not_to respond_to(:build).with(1..2).arguments }
+        it { is_expected.to respond_to(:build).with(0..1).arguments }
+        it { is_expected.to respond_to(:build).with(2..3).arguments }
+        it { is_expected.to respond_to(:build).with(0..3).arguments }
+      end
+      """
+    When I run `rspec respond_to_matcher_argument_range_checking_spec.rb`
+    Then the output should contain all of these:
+      | 8 examples, 4 failures                                          |
+      | expected my_object not to respond to :build with 1..2 arguments |
+      | expected my_object to respond to :build with 0..1 arguments     |
+      | expected my_object to respond to :build with 2..3 arguments     |
+      | expected my_object to respond to :build with 0..3 arguments     |
+
+  @skip-when-splat-args-unsupported
```

Comment:
This tag mentions splat arguments, but as far as I know that's always supported, and you're checking keyword arguments, can you explain further?